django_dbq/management/commands/queue_depth.py

```diff
@@ -7,14 +7,17 @@ class Command(BaseCommand):
     help = "Print the current depth of the given queue"
 
     def add_arguments(self, parser):
-        parser.add_argument("queue_name", nargs="?", default="default", type=str)
+        parser.add_argument("queue_name", nargs="*", default=["default"], type=str)
 
     def handle(self, *args, **options):
-        queue_name = options["queue_name"]
+        queue_names = options["queue_name"]
         queue_depths = Job.get_queue_depths()
 
         self.stdout.write(
-            "queue_name={queue_name} queue_depth={depth}".format(
-                queue_name=queue_name, depth=queue_depths.get(queue_name, 0),
+            " ".join(
+                [
+                    f"{queue_name}={queue_depths.get(queue_name, 0)}"
+                    for queue_name in queue_names
+                ]
```

Comment:
I feel like if we're going to use this for cloudwatch metrics, we'll need a way to identify these log lines more unambiguously. Maybe write out something like `"Queue depths: default=0"` or `event=queue_depths: default=0` or similar? (the latter would be better for us but maybe less "friendly"? Not sure..)

Reply:
i think the latter is probably better for everyone, it's still human readable and more easily machine readable than the former